attendees.php

```diff
@@ -246,7 +257,14 @@
         $table = new html_table();
         $table->head = [get_string('name')];
         $table->align = ['left'];
-        $table->size = ['100%'];
+        $table->size = ['70%'];
+
+        $showusername = facetoface_should_attendees_show_usernames();
+        if ($showusername) {
+            $table->head[] = get_string('username');
+            $table->align[] = 'left';
+            $table->size[] = '30%';
```

Comment:
Does this leave things in a weird state where the width is only 70% if `$showusername` is false?